odoo/addons/base/views/res_partner_views.xml

```diff
@@ -218,11 +218,11 @@
                                     <field name="color"/>
                                     <field name="type"/>
                                     <templates>
-                                        <t t-name="kanban-card" class="row g-0">
-                                            <aside class="col-3 col-sm-2 col-md-3 o_kanban_aside_full">
-                                                <field name="avatar_128" widget="image" options="{'img_class': 'object-fit-cover'}" alt="Contact image"/>
+                                        <t t-name="kanban-card" class="flex-row">
+                                            <aside class="o_kanban_aside_full d-flex flex-shrink-0">
+                                                <field name="avatar_128" class="align-self-center w-100" widget="image" options="{'img_class': 'object-fit-cover'}" alt="Contact image"/>
                                             </aside>
-                                            <main class="col-9 col-sm-10 col-md-9 ps-2">
+                                            <main class="flex-grow-1 w-100 ps-2 ps-md-0">
```

Comment:
Here as well,  like in the "products" template. I'd suggest:
- `<main>`: remove `flex-grow-1 w-100`
- `<aside>`:  remove `flex-shrink-0`
- css: unsure that `main` is `flex-grow: 1` globally

ps: tested briefly in the browser, seems like this setup makes `d-flex`  and `align-self-center` classes unnecessary for `aside` and its field. Could you check?

Reply:
> `<aside>` : remove `flex-shrink-0`

Same as [the last comment](https://github.com/odoo/odoo/pull/169790#discussion_r1652747861), I don't think we should remove it :).

> ps: tested briefly in the browser, seems like this setup makes d-flex and align-self-center classes unnecessary for aside and its field. Could you check?

I've made some tests, these classes are necessary to keep the vertical alignment of `aside`.

Reply:
> Same as https://github.com/odoo/odoo/pull/169790#discussion_r1652747861, I don't think we should remove it :).

Replied there  :) 

> these classes are necessary to keep the vertical alignment of aside.

Not on mobile ;)
These rules seems needed when "aside full" is active only... if confirmed  we can added them in the css directly.  

| with `d-flex` and `align-self-center` | without |
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/8558351/7552c558-57f7-453b-bba5-e53bf93dd8dc) | ![image](https://github.com/odoo/odoo/assets/8558351/f9343c98-e162-4b0b-9840-568215f69100) |

Reply:
> Not on mobile ;)
These rules seems needed when "aside full" is active only... if confirmed we can added them in the css directly.

I've kept it to achieve the same result as before the introduction of `cols` in cards.
But as discussed, centering the image doesn't really make sense. So I've totally removed the centered alignment.
What do you think ?